src/README.md

```diff
@@ -196,6 +196,55 @@ int main(int argc, char* argv[]) {
 }
 ```
 
+### Performance Metrics
```

Comment:
When it gets merged, please, open another PR adding it to C++ and Python docstrings.

Reply:
Done https://github.com/openvinotoolkit/openvino.genai/pull/713